### PR TITLE
 Fix Command Palette not allowing empty alias

### PR DESCRIPTION
**Pull request by FabienTaillon**
### What does this PR do?
* Fix Command Palette not allowing empty alias
* Updated Japanese translation for #1876 (thanks @shunkosa for the updated translation)

### What issues does this PR fix or reference?
* #1929 
* @W-7147786@

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts [hunk 1]
```diff
@@ -286,7 +286,7 @@ export const messages = {
   empty_components: '利用できるコンポーネントがありません',
   error_auth_token: '認証トークン更新中にエラーが発生しました。',
   error_no_org_found: '組織の認証情報が見つかりませんでした。',
-  error_invalid_org_alias: '別名にはアンダースコアと英数字のみを使用できます。',
+  error_invalid_org_alias: '別名にはアンダースコア、スペースと英数字のみを使用できます。',
```
Comment (brpowell) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts]: @ntotten how can we validate this localization change?

Comment (FabienTaillon) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts]: FYI I asked to @shunkosa the updated translation. He is the one that created these translations at first.

Comment (brpowell) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts]: Ah awesome thanks!

**Comment by brpowell** [comment]
Merged this fix as a part of #1953 and it will go out in this week's release, thank you @FabienTaillon and @shunkosa

**Pull request closed without merging**